itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestDatabaseTableDefault.java

```diff
@@ -140,7 +140,7 @@ private void dropTables() throws Exception {
     public void tearDown() throws Exception {
         try {
             if (d != null) {
-                dropTables();
+                //dropTables();
```

Comment:
Why is this change needed?

If it is really needed then the code should be removed; it is generally best to avoid leaving code commented out.

Reply:
A few tests using drop table statements before creating the database resulted in 'Database does not exist' error with the last patch. Hence used DROP DATABASE CASCADE instead of dropping the table first and dropping the DB. Anyways, as we decided to silently return OK in case of non-existent DB, will revert these tests.